src/functions/memo-management/trello-cards/comment-card/handler.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { TrelloService } from "src/database/services/trello-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Handler for creating a comment on a Trello card
+ * 
+ * @param event event
+ * @returns JSON response
+ */
+const createCommentHandler: APIGatewayProxyHandler = async (event: any) => {
```

Comment:
Need the event to be typed, according to what we are expecting to be sent to the lambda